shared/src/models/referentielEngagementEntreprise.model.ts

```diff
@@ -16,6 +16,8 @@ export const ZReferentielEngagementEntreprise = z
     siret: z.string(),
     engagement: z.enum(["handicap"]),
     sources: z.array(z.enum([EntrepriseEngagementSources.FRANCE_TRAVAIL, EntrepriseEngagementSources.LBA, EntrepriseEngagementSources.LES_ENTREPRISE_S_ENGAGENT])),
+    createdAt: z.date(),
+    updatedAt: z.date(),
```

Comment:
```suggestion
    created_at: z.date(),
    updated_at: z.date(),
```

Dans la codebase on peut y trouver les deux formes (malheureusement), mais si on part du principe que nos champs composés dans les modèles de données sont en snake_case, on doit s'efforcer de l'appliquer partout et on modifiera les autres en conséquence quand on aura la bande passante